tremor/input/input_subsystem.py

```diff
@@ -8,11 +8,12 @@
 inputs = {'mouse': [0, 0]}
 
 def init():
+    clamp = lambda value, min_value, max_value: min(max(value, min_value), max_value)
     imgui_renderer = graphics_subsystem.imgui_renderer
     window = graphics_subsystem.window
 
     def mouseclick_callback(window, button, action, modifiers):
-        pass
+        print('fps: %s'%client_main.debug_fps)
```

Comment:
Probably can get rid of this. Renderdoc displays better useful debug information